libmachine/provision/utils.go

```diff
@@ -208,6 +209,14 @@ func matchNetstatOut(reDaemonListening, netstatOut string) bool {
 	return false
 }
 
+func getFilesystemType(p Provisioner, directory string) (fstype string, err error) {
```

Comment:
Nit: Can you please change the named return parameters to being explicitly defined in the function body?  That's convention in this codebase.